xml/http_cookies.xml

```diff
@@ -681,6 +681,14 @@
     <param pos="0" name="service.certainty" value="0.5"/>
   </fingerprint>
 
+  <fingerprint pattern="^laravel_session=">
+    <description>Laravel PHP web application framework</description>
+    <example>laravel_session=eyJpdiI6IjZmcXM4SXZqNVQxY2U2UDVEb1V0SFE9PSIsInZhbHVlIjoiQlRUYXk2WEsxUkZmVU5lRXpBTlZkUURkUnphR0RGMTdaRDkxWHFLVGhtaUZwM0plaWR2aVMyeVQ5MmFzNlVFczVQUExpeDdhUUZveVlCOVUxMVA4dHp6ZUQrK0t3KzBUaVlxb2o4WUxleE1rb29GTlozZWF6cXpUUTRnNFpJTHUiLCJtYWMiOiI3ZGNhNzhmOGRmMWFmZmE5ZjFiYjQ5ODBlNGEwMjgzZWNiZGI2ZTI4ZjFmYWJkMmIxZTgwYzM2NjA1OThlNGNlIn0%3D; expires=Mon, 13-Mar-2023 18:21:13 GMT; Max-Age=7200; path=/; httponly; samesite=lax</example>
+    <param pos="0" name="service.vendor" value="laravel"/>
+    <param pos="0" name="service.product" value="laravel"/>
```

Comment:
I grabbed this from:

> Version 2.3: cpe:2.3:a:laravel:laravel:8.4.2:*:*:*:*:*:*:*
> https://nvd.nist.gov/products/cpe/detail/62651740-3F6C-410D-9C06-9F71B0D077E5?namingFormat=2.3&orderBy=CPEURI&keyword=cpe%3A2.3%3Aa%3Alaravel&status=FINAL%2CDEPRECATED

Interestingly I also found there is CPE for `framework` as the product, but I'm not sure why both co-exist:

> Version 2.3: cpe:2.3:a:laravel:framework:8.4.0:*:*:*:*:*:*:*
> https://nvd.nist.gov/products/cpe/detail/92B54D53-6C45-4877-BBD6-9885AB6FC19F?namingFormat=2.3&orderBy=CPEURI&keyword=cpe%3A2.3%3Aa%3Alaravel&status=FINAL%2CDEPRECATED

Reply:
Also, verify that you are not looking at deprecated entries.

Reply:
Thanks! I've looked at this again, and I believe this is not deprecated, and is the one I want 🤞